actionpack/lib/action_controller/metal/redirecting.rb

```diff
@@ -227,7 +227,7 @@ def _compute_redirect_to_location(request, options) # :nodoc:
     #     url_from("http://example.com/profile")  # => "http://example.com/profile"
     #     url_from("http://evil.com/profile")     # => nil
     #
-    # Subdomains are considered part of the host:
+    # Subdomains are not considered part of the same host:
```

Comment:
The note _**`Subdomains are considered part of the host`**_ just means the app’s subdomain is included when checking the host. So `url_from` will verify that the URL matches the same host (including subdomains). Even other subdomains of the same main domain are considered different host and it makes sense. 

I think adding not could be misleading for readers. Instead, we can make the wording more explanatory. But, keeping as-is is also not a bad idea.